homeassistant/components/airtouch5/__init__.py

```diff
@@ -22,7 +23,10 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     client = Airtouch5SimpleClient(host)
 
     # Connect to the API
-    await client.connect_and_stay_connected()
+    try:
+        await client.connect_and_stay_connected()
+    except TimeoutError as t:
```

Comment:
is the timeout error that this function throws `asyncio.TimeoutError` rather than this one?

Reply:
It did the right thing in my testing (HA tried to restart the integration periodically), so I assume it's the right one.

Reply:
This is fine. See https://docs.python.org/3/library/asyncio-exceptions.html#asyncio.TimeoutError

Reply:
ah thanks!  I see I was playing around with this in python 3.10 so missed that